packages/data-point/src/ReducerObject.js

```diff
@@ -1,82 +1,75 @@
-const isPlainObject = require("lodash/isPlainObject");
 const set = require("lodash/set");
 const cloneDeep = require("lodash/cloneDeep");
 
-const { ReducerNative } = require("./ReducerNative");
+const { Reducer } = require("./Reducer");
+const isReducerConstant = require("./is-reducer-constant");
+
+function isPlainObject(obj) {
+  return (
+    typeof obj === "object" && Object.getPrototypeOf(obj) === Object.prototype
```

Comment:
if the prototype is null, I think it should also return true, in case the object was created with `Object.create(null)`

https://davidwalsh.name/object-create-null

Reply:
do we have a case where we would want to accept an object that was created with `Object.create(null)` ?

Reply:
not really, other than the reason(s) listed in that article. I don't think we should prevent them being used though, especially if these objects are use-defined. An object with a null prototype is even more plain than an object that has one :)

Reply:
I would rather keep away from this situation, it would be supporting one more quirk of the language and a use case that we don't know if it is needed yet. 

As per our documentation we instruct users to use plain objects created through `{...}`. Leaving this out would be one less thing to check for and one less tests to add. 

@csarnataro  thoughts on this?

Reply:
if we're trusting people to follow the instructions, then why even bother calling `isPlainObject`? what are the non-plain objects that might get passed to the ReducerObject constructor?

Reply:
isPlainObject is used in two places, one to check if the reducer being passed is in fact a plain object, and second is when doing the deep crawling of properties

on the first place its needed otherwise we would have no way of knowing if the object being passed matches this type of reducer

Reply:
to rephrase the question, I'm curious why we use `isPlainObject` vs a more generic `isObject` function.

Reply:
my intention behind the is to prevent from any other form of object other than an object created through `{}` , otherwise I would have to test against array, function and regex